scan-triggers/aws-typescript-bucket-full-scan/README.md

```diff
@@ -0,0 +1,51 @@
+# Bucket Full Scan
+
+This is a CDK project to deploy all the required resources to trigger a full scan on a S3 bucket leveraging an existing Trend Micro File Storage Security deployment.
```

Comment:
We should add a warning that if there are a lot of objects in the bucket, there will be the same number of scan request and could flood the Lambdas. And provide our [scaling guide](https://cloudone.trendmicro.com/docs/file-storage-security/performance-scaling/.